packages/tradershub/src/features/cfd/modals/DxtradePasswordModal/DxtradePasswordModal.tsx

```diff
@@ -21,7 +21,8 @@ const DxtradePasswordModal = () => {
     const { error, isLoading, isSuccess, mutate, status } = useCreateOtherCFDAccount();
     const { isSuccess: dxtradeAccountListSuccess } = useDxtradeAccountsList();
     const { data: activeTrading } = useActiveTradingAccount();
-    const { hide } = Provider.useModal();
+    const isDemo = activeTrading?.is_virtual;
```

Comment:
Better way of writing this and imo more readable
```suggestion
    const { is_virtual: isDemo } = activeTrading;
```

Reply:
This code will throw an error if `is_virtual` does not exist on `activeTrading`, ie while the hook is still fetching data and we're trying to destructure it which is why I opted for this method of writing